python/cli/prelude_cli/views/iam.py

```diff
@@ -18,12 +18,12 @@ def iam(ctx):
 @iam.command('create-account')
 @click.pass_obj
 @handle_api_error
-@click.confirmation_option(prompt='Overwrite local account credentials for selected profile?')
-def register_account(controller):
+@click.option('-b', '--bypass', help='bypass email verification', is_flag=True, default=False)
+def register_account(controller, bypass):
     """ Register a new account """
     handle = click.prompt('Enter a handle')
     with Spinner():
-        data = controller.new_account(handle=handle)
+        data = controller.new_account(handle=handle, bypass=bypass)
     print_json(data=data)
     print("\nCheck your email to verify your account.\n")
```

Comment:
Let's adjust the msg here, to not direct the user to check their email if bypassing. While you're in there, change the click option from handle to email.